### PR TITLE
Update sandman_de.ts

### PR DESCRIPTION
**Pull request by bastik-1001**
Fresh fork. New strings translated. No complaints about spelling mistakes, beside words that are made up, like live channel.

### CODE REVIEW

**Comment by isaak654** [comment]
41 additions and you rushed the pull request title, I think an additional review could be useful.
@APMichael

**Comment by bastik-1001** [comment]
I expected Github to pick the pull request title from the only file that has been changed, but did not check if it picked that up.

Yeah, having an additional review is indeed useful. Whatever changes might be proposed, please accept them.

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 1]
```diff
@@ -278,97 +278,97 @@
...
         <location filename="OnlineUpdater.cpp" line="391"/>
         <source>&lt;p&gt;Do you want to download the updates?&lt;/p&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;p&gt;Möchten Sie die Updates runterladen?&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="OnlineUpdater.cpp" line="393"/>
         <source>&lt;p&gt;Do you want to go to the &lt;a href=&quot;%1&quot;&gt;update page&lt;/a&gt;?&lt;/p&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;p&gt;Möchten Sie zur&lt;a href=&quot;%1&quot;&gt;Updateseite gehen&lt;/a&gt;?&lt;/p&gt;</translation>
```
Comment (APMichael) [inline SandboxiePlus/SandMan/sandman_de.ts:291]: Missing space between "zur" and "\&lt;".

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 2]
```diff
@@ -278,97 +278,97 @@
...
         <location filename="OnlineUpdater.cpp" line="734"/>
         <source>Failed to download installer from: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Fehler beim Runterladen des Installers von: %1</translation>
     </message>
     <message>
         <location filename="OnlineUpdater.cpp" line="760"/>
         <source>&lt;p&gt;A new Sandboxie-Plus installer has been downloaded to the following location:&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;%2&quot;&gt;%1&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Do you want to begin the installation? If any programs are running sandboxed, they will be terminated.&lt;/p&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;p&gt;Ein neuer Sandboxie-Plus Installer wurde an folgenden Speicherort runtergeladen:&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;%2&quot;&gt;%1&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Möchten Sie mit der Installation beginnen? Falls Programme in einer Sandbox laufen, werden diese beendet..&lt;/p&gt;</translation>
```
Comment (APMichael) [inline SandboxiePlus/SandMan/sandman_de.ts:371]: At the end of the sentence (after "beendet") is an additional unnecessary dot.

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 3]
```diff
@@ -6664,47 +6664,47 @@ Bitte beachten Sie, dass diese Werte aktuell nutzerspezifisch sind und global f
     <message>
         <location filename="Forms/SettingsWindow.ui" line="1110"/>
         <source>New full versions from the selected release channel.</source>
-        <translation type="unfinished"></translation>
+        <translation>Neue vollständige Updates werden vom ausgewählten Veröffentlichungskanal ausgewählt.</translation>
```
Comment (APMichael) [inline SandboxiePlus/SandMan/sandman_de.ts:6667]: Here I am not sure, possibly the following sentence would fit better:
"Neue vollständige Versionen aus dem ausgewählten Veröffentlichungskanal beziehen."

**Comment by APMichael** [comment]
> 41 additions and you rushed the pull request title, I think an additional review could be useful. @APMichael

Looks good to me, except for the few minor corrections needed.

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 4]
```diff
@@ -6664,47 +6664,47 @@ Bitte beachten Sie, dass diese Werte aktuell nutzerspezifisch sind und global f
...
         <location filename="Forms/SettingsWindow.ui" line="1222"/>
         <source>Search in the Live channel</source>
-        <translation type="unfinished"></translation>
+        <translation>Suche im Livekanal</translation>
     </message>
     <message>
         <location filename="Forms/SettingsWindow.ui" line="1229"/>
         <source>The preview channel contains the latest GitHub pre-releases.</source>
-        <translation type="unfinished"></translation>
+        <translation>Der Vorschaukanal enthält die aktuellsten Github-Vorabveröffentlichungen.</translation>
```
Comment (APMichael) [inline SandboxiePlus/SandMan/sandman_de.ts:6697]: Correct spelling: "GitHub"

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 5]
```diff
@@ -6664,47 +6664,47 @@ Bitte beachten Sie, dass diese Werte aktuell nutzerspezifisch sind und global f
...
     </message>
     <message>
         <location filename="Forms/SettingsWindow.ui" line="1293"/>
         <source>The stable channel contains the latest stable GitHub releases.</source>
-        <translation type="unfinished"></translation>
+        <translation>Der Stabilkanal enthält die aktuellsten stabilen Github-Veröffentlichungen.</translation>
```
Comment (APMichael) [inline SandboxiePlus/SandMan/sandman_de.ts:6702]: Correct spelling: "GitHub"

**Comment by APMichael** [comment]
> A few hours ago I updated some english strings, so in case there are conflicts in the .ts file in this PR: ...

Are you sure about the changes from "update" to "updated"?

I think it was correct. Examples:
"failed to download update informations" means "failed to download information about the update"
"failed to load update json file" means "failed to load JSON file of the update"
"update signature is invalid !!!" means "signature of the update is invalid !!!"

**Comment by isaak654** [comment]
> Are you sure about the changes from "update" to "updated"?

It was also correct before, but I think it is more common and understandable in the current way.
If @DavidXanatos does not agree, he is free to restore whatever he wants.

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 6]
```diff
@@ -278,97 +278,97 @@
...
         <location filename="OnlineUpdater.cpp" line="540"/>
         <source>failed to download a particular file</source>
-        <translation type="unfinished"></translation>
+        <translation>Fehler beim Runterladen einer bestimmten Datei</translation>
     </message>
     <message>
         <location filename="OnlineUpdater.cpp" line="541"/>
         <source>failed to scan existing installation</source>
-        <translation type="unfinished"></translation>
+        <translation>Fehler beim Scannen der bestehenden Installation</translation>
```
[the remark this answers is not in the document]
Comment (bastik-1001) [reply SandboxiePlus/SandMan/sandman_de.ts]: That's how it probably should be, but I had kept what it actually said. Now I changed it to what you suggested, which fits much better.

File: SandboxiePlus/SandMan/sandman_de.ts (hunk 1 above)
Comment (bastik-1001) [reply SandboxiePlus/SandMan/sandman_de.ts]: Fixed.

File: SandboxiePlus/SandMan/sandman_de.ts (hunk 2 above)
Comment (bastik-1001) [reply SandboxiePlus/SandMan/sandman_de.ts]: Fixed. Thanks,

File: SandboxiePlus/SandMan/sandman_de.ts (hunk 3 above)
Comment (bastik-1001) [reply SandboxiePlus/SandMan/sandman_de.ts]: I went with your wording, which is easier to understand.

File: SandboxiePlus/SandMan/sandman_de.ts (hunk 4 above)
Comment (bastik-1001) [reply SandboxiePlus/SandMan/sandman_de.ts]: Changed.

File: SandboxiePlus/SandMan/sandman_de.ts (hunk 5 above)
Comment (bastik-1001) [reply SandboxiePlus/SandMan/sandman_de.ts]: Changed.

**Comment by bastik-1001** [comment]
Something went wrong.

**Comment by isaak654** [comment]
I see, thanks for trying.

**Pull request closed without merging**